Command/RequestPasswordResetHandler.php

```diff
@@ -46,10 +46,6 @@ public function handle(RequestPasswordDataTransferObject $dataTransferObject)
     {
         $user = $this->userRepositoryCollection->findUserByUserName($dataTransferObject->userName);
 
-        if ($user === null) {
-            throw new EntityNotFoundException();
```

Comment:
Ja oke maar zoals ik al zei ga je dan op zoveel plaatsen die `@throws blablabla` moeten gaan zetten dat het gewoon de boel onduidelijker maakt. Het is niet van zulk belang dat je overal moet weten dat die exceptions kunnen voorkomen he. PHPDoc is er om te helpen, niet om te dicteren wat er gebeurt.